servers/cromwell/jobs/controllers/utils/task_statuses.py

```diff
@@ -26,6 +27,7 @@
 
 CROMWELL_EXECUTION_STATUS_MAP = {
     CromwellExecutionStatus.NOTSTARTED: ApiStatus.SUBMITTED,
+    CromwellExecutionStatus.WAITINGFORQUEUESPACE: ApiStatus.SUBMITTED,
```

Comment:
Please update the docstring below as well to include this status: https://github.com/DataBiosphere/job-manager/blob/e49012d6a068a9989e6b88b249f1cbdb1903e51d/servers/cromwell/jobs/controllers/utils/task_statuses.py#L47

Reply:
Good catch! will do